test/system/030-run.bats

```diff
@@ -1233,4 +1233,18 @@ search               | $IMAGE           |
     done < <(parse_table "$tests")
 }
 
+@test "podman run umask" {
```

Comment:
we already have tests for umask with podman run, the important part is to test it with exec so the test should mention that and moved to 075-exec.bats

Reply:
Any idea why this is not working on Debian? I tried with runc, and that worked fine.

Reply:
There are no system tests for --umask that I see.